db/migrations/postgres/000100_allow_null_listener_location.up.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
```

Comment:
Probably need a SQLite migration (which would be a full table copy since you can't drop a constraint) _or_ if that's too heavy, need to edit the creation of the original SQLite table creation to not include NOT NULL in the first place (and note migrations won't be supported).

Reply:
thank you, not sure how I forgot that 😅